spark/src/main/java/org/apache/iceberg/spark/source/PartitionKey.java

```diff
@@ -48,17 +48,16 @@
   private final Accessor<InternalRow>[] accessors;
 
   @SuppressWarnings("unchecked")
-  PartitionKey(PartitionSpec spec) {
+  PartitionKey(PartitionSpec spec, Schema inputSchema) {
     this.spec = spec;
-
```

Comment:
Can you add back these newlines? We try to avoid non-functional changes like this because they can cause commit conflicts.